### PR TITLE
U03: A new player is only able to join in the lobby 

### PR DESCRIPTION
**Pull request by Croxsy**
added an if statement into join, that if the server doesnt return null then the playerId and everything is saved. Otherwise it returns (client goes then to titlescreen)

### CODE REVIEW

File: src/managers/SessionManager.ts [hunk 1]
```diff
@@ -71,8 +71,11 @@ class SessionManager {
             "/join",
             requestBody,
         );
-        PlayerManager.saveId(response.data);
-        location.pathname = "";
+        if (response !== null) {
```
Comment (C0DECYCLE) [inline src/managers/SessionManager.ts:74]: does it work like this? is the response null or the response body?

Comment (C0DECYCLE) [reply src/managers/SessionManager.ts:74]: i tested it, response body is empty so you need to change the code

Comment (C0DECYCLE) [reply src/managers/SessionManager.ts:74]: i would do it like this:
```
        if (!response.data) {
            return;
        }
        PlayerManager.saveId(response.data);
        location.pathname = "";
```

File: src/managers/SessionManager.ts [hunk 2]
```diff
@@ -71,8 +71,11 @@ class SessionManager {
...
+            PlayerManager.saveId(response.data);
+            location.pathname = "";
+        }
+        return;
```
Comment (C0DECYCLE) [inline src/managers/SessionManager.ts:78]: An empty return at the end of a function isn't required.

**Review by C0DECYCLE — CHANGES_REQUESTED** [review]

**Review by paul891112 — APPROVED** [review]
Nothing to add besides that

**Review by C0DECYCLE — APPROVED** [review]
LGTM

**Pull request merged**